poseidon/src/pasta/fp_kimchi.rs

```diff
@@ -830,7 +830,7 @@ pub fn params() -> ArithmeticSpongeParams<Fp> {
 }
 
 /// the fp sponge params
-pub fn static_params() -> &'static ArithmeticSpongeParams<Fp> {
-    static PARAMS: Lazy<ArithmeticSpongeParams<Fp>> = Lazy::new(params);
+pub fn static_params() -> &'static Params {
+    static PARAMS: Lazy<Params> = Lazy::new(params);
```

Comment:
`Lazy` is actually no longer required here because there are no allocations. I didn't change this though as I'm sure the compiler can handle optimizing that part away. This should be fixed in a follow up